### PR TITLE
Added two scripts to change the wallpaper

### PR DESCRIPTION
**Pull request by juacq97**
The one named `nitrogen` allow to change the wallpaper using
nitrogen. The one named `pywal` allows to change the wallpaper and the
terminal color scheme using `pywal`.
Both of them need to put the cursor over an image, there's no need to
select the file with space.

### CODE REVIEW

**Comment by jarun** [comment]
Please rename the file `nitrogen` to `nwal` (short and sweet).

File: user-scripts/README.md [hunk 1]
```diff
@@ -8,9 +8,11 @@
 | imgur | Upload an image file to imgur |
 | kdeconnect | Send selected files to Android device using kdeconnect-cli |
 | ndiff | File and directory diff for selection using vimdiff |
+| nitrogen | Set the selected image as wallpaper using nitrogen |
```
Comment (jarun) [inline user-scripts/README.md]: Please rename the file to `nwal`. Note that the entry would move below `nlaunch` alphabetically with that.

File: user-scripts/nitrogen [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```
Comment (jarun) [inline user-scripts/nitrogen]: Please try to make it work with just `sh`? We try to stay `bash`-independent.

File: user-scripts/pywal [hunk 3]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```
Comment (jarun) [inline user-scripts/pywal]: Please try to make it work with just `sh`.

File: user-scripts/nitrogen [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+# Description: Set the selected image as wallpaper using nitrogen. Just put the cursor on a image and run the script. 
+#
+# Shell: POSIX Compliant
+# Author: juacq97
+
+if ! [ -z "$1" ]; then
+    if [ $(mimetype --output-format %m $1 | awk -F '/' '{print $1}') == "image" ]; then
+	nitrogen --set-zoom-fill --save $1
+	notify-send -a "nnn" "Wallpaper changed!"
```
Comment (jarun) [inline user-scripts/nitrogen]: We need to comment out notify-send in line 11 and line 13 as it's not platform-independent. Please follow the `kdeconnect` script.

File: user-scripts/pywal [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+# Description: Set the selected image as wallpaper using pywal and change the terminal color schemes. Just put the cursor on a image and run the script. 
+#
+# Shell: POSIX Compliant
+# Author: juacq97
+
+if ! [ -z "$1" ]; then
+    if [ $(mimetype --output-format %m $1 | awk -F '/' '{print $1}') == "image" ]; then
+	wal -i $1
+	notify-send -a "nnn" "Wallpaper changed!"
```
Comment (jarun) [inline user-scripts/pywal]: Same goes for notify-sends int his script.

File: user-scripts/nwal [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+#
+# Shell: POSIX Compliant
+# Author: juacq97
+
+if ! [ -z "$1" ]; then
+    if [ $(mimetype --output-format %m $1 | awk -F '/' '{print $1}') == "image" ]; then
+	nitrogen --set-zoom-fill --save $1
+	notify-send -a "nnn" "Wallpaper changed!"
+    else
+	notify-send -a "nnn" "You need to select a image"
```
Comment (jarun) [inline user-scripts/nwal]: You need to select **an** image.

File: user-scripts/pywal [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+    else
+	notify-send -a "nnn" "You need to select a image"
```
Comment (jarun) [inline user-scripts/pywal]: You need to select **an** image.

File: user-scripts/nwal (hunk 6 above)
Comment (jarun) [reply user-scripts/nwal]: How about the message "No image selected"

Comment (juacq97) [reply user-scripts/nwal]: "No image selected" sounds better

Comment (jarun) [reply user-scripts/nwal]: Cool. Please use that string.

**Comment by jarun** [comment]
Thank you for your contribution!

File: user-scripts/nwal (hunk 6 above)
Comment (juacq97) [reply user-scripts/nwal]: Sorry for the misspellings, English is not my native language and I miss some things

Comment (jarun) [reply user-scripts/nwal]: No problem at all!

**Comment by jarun** [comment]
If you have the time, can you see if it's possible to come up with a script that can show images using `ueberzug` (single instance) and then copy the name of the last file opened in the viewer to the system clipboard?

People want the image preview capability but I would like to offload the capability to a distinct script rather than within `nnn`.

**Comment by jarun** [comment]
There is a similar project for `vifm`: https://github.com/cirala/vifmimg

See if that helps.

**Comment by juacq97** [comment]
I'll see what I can do, if I get a functional script I'll do a pull request

**Comment by jarun** [comment]
That's very kind of you. Thanks!

**Pull request merged**